packages/strapi-plugin-users-permissions/services/UsersPermissions.js

```diff
@@ -22,7 +22,7 @@ const DEFAULT_PERMISSIONS = [
     type: 'users-permissions',
     roleType: 'public',
   },
-  { action: 'changepassword', controller: 'auth', type: 'users-permissions', roleType: 'public' },
+  { action: 'resetPassword', controller: 'auth', type: 'users-permissions', roleType: 'public' },
```

Comment:
@alexandrebodin I know this is merged already, but this seems like will not work as one might expected. The correct action name is `resetpassword`. 🤔